packages/progressbar/README.md

```diff
@@ -1,18 +1,18 @@
 # ProgressBar
 
-A ProgressBar is a linear chart that can be used used to show parts of a whole or to represent the completion of a task.
+The `ProgressBar` is a linear chart that can be used to show parts of a whole or to represent the completion of a task.
 
 ## Text content
 
 ### Caption
 
-Sometimes a ProgressBar requires a text caption to show what data it's measuring. A caption is only needed if the ProgressBar is lacking context in the UI. For example, a progress bar would not need a caption if it's in a table column that has a labeled header.
+Sometimes a ProgressBar requires a text caption to show what data it's measuring. A caption is only needed if the ProgressBar is lacking context in the UI. For example, a ProgressBar would not need a caption if it's in a table column that has a labeled header.
 
 By default, this text should appear above the ProgressBar on the right side. This is where the text appears when it's passed to the `caption` prop.
 
 ### Value
 
-If it's important that users know the exact value of what's being measured, it's best to include text that shows exactly what the value is. For example, if I have a chart that's showing 45% of some total, I should display "45%" near the chart.
+If it's important that users know the exact value of what's being measured, it's best to include text that shows exactly what the value is. For example, if you have a chart that's showing 45% of some total, you should display "45%" near the chart.
```

Comment:
second sentence you could drop the second person altogether: "For example, the text "45%" should accompany a chart showing 45% of some total."